gitbutler-ui/src/lib/components/ProjectAvatar.svelte

```diff
@@ -14,32 +14,24 @@
 		'#5FD2B0'
 	];
 
-	const stringToGradient = (string: string | undefined) => {
-		if (!string) {
-			return `linear-gradient(45deg, ${gradients[0][0]} 15%, ${gradients[0][1]} 90%)`;
+	function nameToColor(name: string | undefined) {
+		const trimmed = name?.replace(/\s/g, '');
+		if (!trimmed) {
+			return `linear-gradient(45deg, ${colors[0][0]} 15%, ${colors[0][1]} 90%)`;
```

Comment:
@PavelLaptev I forgot to ask about this, I don't see under what circumstances the parameter to this function can be `undefined`?